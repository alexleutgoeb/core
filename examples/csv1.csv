a;b;c
1;x;h\"ello\\ world
