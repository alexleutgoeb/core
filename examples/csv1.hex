q(I,X,Z,Y) :- p(I,X,Y,Z).
