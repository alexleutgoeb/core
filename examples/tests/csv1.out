a;c;b
1;h\"ello\\ world;x
